puppet/zulip/manifests/profile/smokescreen.pp

```diff
@@ -0,0 +1,57 @@
+# @summary Outgoing HTTP CONNECT proxy for HTTP/HTTPS on port 4750.
+#
+class zulip::profile::smokescreen {
+  include zulip::profile::base
+  include zulip::supervisor
+
+  $golang_version = '1.14.10'
```

Comment:
@andersk do you think at this point we should make a doc with a master checklist of the third-party things that we have pinned versions of, maybe as a section on `docs/subsystems/release-checklist.md` and/or `docs/subsystems/dependencies.md`, to help make sure we don't lose track of things like this?  I'd hate for us to discover in 2 years that we've been stuck running an insecure version of `Go` because we forget we had it pinned.

(Alternatively, could we just get `goland` from apt, or do we need a specific version?)

Reply:
From https://github.com/stripe/smokescreen/blob/master/README.md#dependencies:
> Smokescreen is built and tested using the following Go releases. Generally, Smokescreen will only support the two most recent Go versions.
>
> go1.14.x
> go1.15.x

And Bionic only ships 1.10:
```
$ apt-cache policy golang
golang:
  Installed: (none)
  Candidate: 2:1.10~4ubuntu1
  Version table:
     2:1.10~4ubuntu1 500
        500 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages
```

I can bump to the latest 1.15; it came out 2020/08/11, after I wrote this commit originally.

Reply:
Actually, I'm going to wait to see if stripe/smokescreen#136 lands and update to 1.16 if so.